frontend/views/reports.js

```diff
@@ -85,6 +133,7 @@ module.exports = Backbone.View.extend({
         this.$el.html(template(context));
         this.resultsView = new ReportTableView({
```

Comment:
This was already this way, but I wonder why we're constructing a whole new view object after fetching the results - couldn't we just call `render()` on the one we have already?